src/User.java

```diff
@@ -0,0 +1,78 @@
+import java.util.Scanner;
+
+
+public class User {
+    private String userName;
+    private String action;
+    private double balance = 100.0;
+    private double withdraw;
+    private String cancel;
+    Scanner input = new Scanner(System.in);
+
+
+    public String getUser() {
+        return userName;
+    }
+    public String getAction() {
+        return action;
+    }
+    public double getBalance() {
+        return balance;
+    }
+    private double getWithdraw() {
+        return withdraw;
+    }
+    public String getCancel() {
+        return cancel;
+    }
+    public void setUser(String user)  {
+        this.userName = user;
+    }
+    public void setBalance(double balance) {
+        this.balance = balance;
+    }
+    public void setWithdraw(double withdraw) {
```

Comment:
Name your methods in a way that makes more grammatical sense.  For this I would have just called it `withdrawal(double withdrawal)`